gluster/block/cmd/glusterblock-provisioner/glusterblock-provisioner.go

```diff
@@ -182,7 +182,7 @@ func (p *glusterBlockProvisioner) Provision(options controller.VolumeOptions) (*
 	// Calculate the size
 	volSize := options.PVC.Spec.Resources.Requests[v1.ResourceName(v1.ResourceStorage)]
 	volSizeBytes := volSize.Value()
-	volszInt := int(util.RoundUpSize(volSizeBytes, 1000*1000*1000))
+	volszInt := int(util.RoundUpToGiB(volSizeBytes))
```

Comment:
We could just use RoundUpSize and reference GiB, instead of creating a new method.

Here is an example - https://github.com/kubernetes-incubator/external-storage/blob/master/local-volume/provisioner/pkg/discovery/discovery.go#L213

Reply:
Sure, we could do that. If that's preferred, I'm happy to change the PR.
I was mainly following the pattern of https://github.com/kubernetes/kubernetes/pull/56581 that introduced the corresponding function in Kubernetes proper.